### PR TITLE
Add lazer faq post

### PR DESCRIPTION
**Pull request by peppy**

### CODE REVIEW

File: _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+- A more streamlined UI and a fresh design
+- Access to most of the web-only content from within the game (news, changelogs, user profiles, rankings, beatmap info)
+- A song density graph in gameplay
+- The ability to seek in replays
+- The ability to hide difficulties
+- Soft delete everywhere – you can restore beatmaps after deleting if you made a mistake
+
+### Will xxx feature be implemented in lazer?
+
+We are primarily working on feature parity, but if the feature you've asked for has been something people have been asking for for quite a will, you can be 90% sure that it is one of the reasons we are working so hard to rewrite osu! in the first place!
```
Comment (Ephemeralis) [inline _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md]:
```suggestion
We are primarily working on feature parity, but if the feature you've asked for has been something people have been asking for for quite a while, you can be 90% sure that it is one of the reasons we are working so hard to rewrite osu! in the first place!
```

File: _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+Yes. We will be 100% supporting skins in gameplay, along with some menu-level interface elements (like the back button). We're currently at about 90-95% implementation of skins for osu! and osu!catch, while osu!mania and osu!taiko need further work.
+
+### Will the new interface be skinnable?
+
+We will be adding more skin support after we're happy with the existing gameplay and support for legacy skins, including the ability to customise more of the menu screens.
+
+### Will there be touch controls to allow playing fast streams etc. on mobile?
+
+Yes, these are in development. For now you can connect a hardware keyboard.
```
Comment (Ephemeralis) [inline _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md]:
```suggestion
Yes, these are in development. For now, you can connect a hardware keyboard.
```

File: _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
+---
+title: A definitive osu!lazer FAQ
+date: 2019-10-21T00:00:00.000Z
+published: true
+---
+
```
Comment (Ephemeralis) [inline _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md]: the absence of a header paragraph even on a FAQ newspost is kinda weird, maybe add a quick blurb about this article explaining frequently asked questions about the osu!lazer project?

File: _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+Along the way, we ran into some serious blocking issues which resulted in the decision to rewrite osu!lazer as a new project from scratch, rather than building on top of the existing osu! codebase. This happened in the early months of 2016.
+
+Since then we have been working extra-hard to build both a [game framework](https://github.com/ppy/osu-framework) and rebuild all the intricacies of osu!, while also fixing all the issues that existed in the old codebase to allow for implementing new features and extending on the game more than ever before.
+
+We hope you can understand how excited we are to have this opportunity. The decision could only be made because we already have such a stable game client which (in general) satisfies the community's needs quite well.
+
+### Why is it taking so long?
+
+We are aiming for perfection. That has meant, on several occasions, writing and rewriting systems until we are content that they will last well into the future. What we are working on is not a temporary solution, rather something to carry the osu! community through the rest of its (hopefully long) life.
```
Comment (Ephemeralis) [inline _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md]: new paragraph after perfection for heavy emphasis

File: _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md (hunk 3 above)
Comment (peppy) [reply _posts/2019/10/2019-10-21-a-definitive-lazer-faq.md]: yup, will add

**Pull request merged**